tests/vmi_multus_test.go

```diff
@@ -1352,21 +1351,30 @@ func configureNodeNetwork(virtClient kubecli.KubevirtClient) {
 	}, time.Minute, time.Second).Should(Equal(len(nodes.Items)))
 }
 
-func checkSriovEnabled(virtClient kubecli.KubevirtClient, sriovResourceName string) bool {
+func validateSRIOVSetup(virtClient kubecli.KubevirtClient, sriovResourceName string, minRequiredNodes int) error {
 	nodes := tests.GetAllSchedulableNodes(virtClient)
 	Expect(nodes.Items).ToNot(BeEmpty(), "There should be some compute node")
 
+	if len(nodes.Items) == 0 {
```

Comment:
What is the difference between this `if` and `Expect(nodes.Items).ToNot(BeEmpty(), "There should be some compute node")`?

Reply:
Ahh, that is a leftover. I will keep the assert for now to limit the change.